draft-kuehlewind-shmoo-online-meeting.md

```diff
@@ -88,34 +88,41 @@ following this 1-1-1 rule to share the pain has/seems to have rough consensus.
 
 ## Time Zone Selection
 
-All fully online IETF plenary meetings begin at 0500 ("Asia"), 1200 ("Europe"), or 2100 ("North America") UTC. The names
-are not meant to imply that all participants in a given region will find the times convenient given their personal
-schedules, but are useful for the selection rules below. These location names are consistent with the venue
-selection criteria in {{RFC8719}}.
-
-The selected slots have been proposed to minimize inconvenience while not excessively penalizing any time zone.
-Effectively, there is an early morning and a late afternoon meeting for two of the three regions in
-each slot. E.g. the "Asia" 0500 UTC slot would be 0600 CET (early morning) and 1300 China Standard Time (afternoon). Since fully
-online meeting days are expected to be shorter then in-person meetings, this slot is roughly within the "usual" working hours
-of both regions.
-
-The intent of rotating between these three slots is to scatter meetings throughout the course of the global day, to maximize the ease
-of participants to occasionally attend regardless of their location and what time of day is optimal for their schedule.
+All fully online IETF plenary meetings begin at 0500 ("North America Night"),
+1400 ("Asia Night"), or 2200 ("Europe Night") UTC. The times are not meant to
```

Comment:
I think we need to consider summer and winter time, otherwise we just cause unnecessary pain. E.g I think in summer time we should use 13 UTC instead of 14 UTC. And in winter we should maybe just avoid using the 22 UTC sot...?

Reply:
alright, I've come up with 5/15/23 in Northern Summer and 6/16/00 for Northern Winter. I'll make the change

Reply:
Really? I thought if you do the evaluation separately it would rather move the summer time earlier. Also why is there a two hour difference?

Reply:
Summer is earlier: summer 0500/winter 0600

For the slots with a 2 hour difference, this is because of India. They don't have DST, and moving times one hour later pushes the second-worst slot into some really bad hours.

Reply:
Obviously the differences in score are not huge with these little tweaks, but 6/16/00 is the minimum I found